### PR TITLE
Fix description of exponentiated quadratic covariance function.

### PR DESCRIPTION
**Pull request by lawrennd**

### CODE REVIEW

File: GPy/kern/src/stationary.py [hunk 1]
```diff
@@ -641,6 +641,32 @@ def dK_dr(self, r):
...
+class Cosine(Stationary):
+    """
+    Cosine Covariance function
+    
+    .. math::
+
+        k(r) = \sigma^2 \cos(r)
+
+    """
     def __init__(self, input_dim, variance=1., lengthscale=None, ARD=False, active_dims=None, name='Cosine'):
```
Comment (ekalosak) [inline GPy/kern/src/stationary.py]: You've defined the `class Cosine(Stationary)` twice.

**Review by ekalosak — CHANGES_REQUESTED** [review]
@lawrennd looks like you defined the Cosine kernel twice - aside from that this LGTM.

File: GPy/kern/src/stationary.py (hunk 1 above)
Comment (lawrennd) [reply GPy/kern/src/stationary.py]: Doh! I'll fix.

Comment (lawrennd) [reply GPy/kern/src/stationary.py]: Fixed!

**Pull request merged**